app/controllers/package_controller.rb

```diff
@@ -133,7 +133,7 @@ def set_categories
       { :name => _("Graphics"), :id => "Graphics", :icon => "brush", :categories => ["Graphics"] },
       { :name => _("Office & Productivity"), :id => "Office", :icon => "document", :categories => ["Office"] },
       { :name => _("Network"), :id => "Network", :icon => "world-outline", :categories => ["Network"] },
-      { :name => _("System & Utility"), :id => "Tools", :icon => "spanner-outline", :categories => ["Settings", "System", "Utility"] },
```

Comment:
Nitpick: Does rubocop complain about that removed `,`? If not I would leave it

Reply:
yes, that is the `Style/TrailingCommaInLiteral` offense.

Reply:
okay, ty for clearing it up @alexandergraul :grin: